test-requirements.txt

```diff
@@ -0,0 +1 @@
+ansible-lint==2.3.6
```

Comment:
Is this the only version of ansible-lint that works or can we use a version range like
``ansible-lint>=2.3.6,<3.0``